.github/workflows/ci.yml

```
@@ -747,45 +747,11 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
-  build-tasklist-backend:
+  tasklist-backend-tests:
```

❓ Should it be called 
```suggestion
  tasklist-docker-tests:
```

---

in the beginning, I named like that, then I decided to rename it as it can be extended to other tasklist backend tests that we can include in the common CI in the future

---

Yeah i know but right now it isnt so i would reflect the current state

---

I'm fine with either name but I'd like it to be consistent with the name of the called workflow, which is currently `tasklist-docker-tests.yml`.

> as it can be extended to other tasklist backend tests that we can include in the common CI in the future

🔧 If you want to have that as a place for future extension, I think it makes sense to have a more generic name for that workflow file.

> would reflect the current state

💭 Could leave a comment in the `ci.yml` indicating the current state + future aspiration aka `# Currently only runs Docker tests but should be extended in future to also include backend tests`

---

I renamed the job to `tasklist-docker-tests` and added a comment to tell that we need to extends it with more IT tests